lighthouse-core/test/runner-test.js

```diff
@@ -392,18 +390,16 @@ describe('Runner', () => {
     const url = 'https://example.com';
     const config = new Config({
       audits: [
-        'is-on-https'
+        'content-width'
       ],
 
       artifacts: {
-        HTTPS: {
-          value: true
-        }
+        ViewportDimensions: '#ffffff'
```

Comment:
it's not testing that the `content-width` didn't throw an error, though, just that the artifacts make it through. Should still update the assertion value to something less confusing for the next reader